alert-policies/ext-proxysql/alert.yml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
```suggestion
```

Reply:
@k1nect this needs to be removed.  Looks like a boilerplate example.